src/bin/saitocli.rs

```diff
@@ -251,18 +277,29 @@ pub async fn main() -> saito_rust::Result<()> {
                     std::process::exit(1);
                 });
 
+        // let input_ordinal: u8 = matches
+        //     .value_of("input-ordinal")
+        //     .unwrap()
+        //     .parse()
+        //     .unwrap_or_else(|_error| {
+        //         println!("input_ordinal must be a float");
+        //         println!("got {}", matches.value_of("input-ordinal").unwrap());
+        //         std::process::exit(1);
+        //     });
+
         let mut transaction = Transaction::new();
         transaction.set_transaction_type(TransactionType::Normal);
 
-        // TODO: get inputs from the wallet and use the amount specified
+        // get inputs from the wallet and use the amount specified
         let mut input1 = Slip::new();
         input1.set_publickey(wallet.get_publickey());
-        input1.set_amount(0);
+        input1.set_amount(amount);
+        // input1.set_slip_ordinal(input_ordinal);
         input1.set_uuid([0; 32]);
```

Comment:
hmmmm, does an input need an amount???

I honestly don't even know. In Bitcoin it's not needed but I guess in Saito it is. This is part of the reason I didn't want a single type for input/output.

Inputs should have a uuid(tx hash) and an ordinal.